packages/app/src/lib/import.ts

```diff
@@ -393,13 +405,15 @@ async function lpParseRow(row: string[]): Promise<VaultItem> {
 export async function asLastPass(file: File): Promise<VaultItem[]> {
     const data = await readFileAsText(file);
     const papa = await loadPapa();
+    const isLegacyFormat =
+        data.split("\n")[0].trim() === "url,username,password,totp,extra,name,grouping,fav" ? false : true;
```

Comment:
```suggestion
    const isLegacyFormat = data.split("\n")[0].trim() === "url,username,password,totp,extra,name,grouping,fav";
```

`data.split("\n")[0].trim() === "url,username,password,totp,extra,name,grouping,fav"` already evaluates to a boolean, so the ternary expression is redundant here.

Reply:
Ah, that makes sense. In that case let's keep it as-is then :)